docs/outlook/contextless.md

```diff
@@ -17,14 +17,64 @@ With a simple manifest configuration, you can create Outlook add-ins for the Mes
 
 ## Set up your environment
 
-Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator) to create an add-in project with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
+- To create an add-in project that uses the unified manifest for Microsoft 365, complete the [Outlook quick start for the unified manifest](../quickstarts/outlook-quickstart-json-manifest.md).
+- To create an add-in project that uses the XML manifest with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md), complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeomangenerator).
 
 To turn on this feature in a preexisting add-in project, see [Configure the manifest](#configure-the-manifest).
 
 ## Configure the manifest
 
-> [!NOTE]
-> This feature isn't currently supported in the [Unified manifest for Microsoft 365 (preview)](../develop/json-manifest-overview.md), but the team is working on making this available.
+# [Unified manifest for Microsoft 365](#tab/jsonmanifest)
+
+1. In your preferred code editor, open the Outlook quick start project that you created in the quick start [Outlook quick start for the unified manifest](../quickstarts/outlook-quickstart-json-manifest.md).
+
+1. Open the manifest.json file located at the root of the project.
+
+1. In the first object in the "extensions.runtimes" array, do the following:
+
+    1. Change the "requirements.capabilities.minVersion" to "1.13".
+    1. Add a "supportsNoItemContext" property to the object in the "actions" array and set its value to `true`.
+    1. Add a "multiselect" property to the same object and set it to `true`.
+    1. Change the "pinnable" property in the same object to `true`.
+
+    When you are done, it should look like the following.
+
+    ```json
+    "runtimes": [
+        {
+            "requirements": {
+                "capabilities": [
+                    {
+                        "name": "Mailbox",
+                        "minVersion": "1.13"
+                    }
+                ]
+            },
+            "id": "TaskPaneRuntime",
+            "type": "general",
+            "code": {
+                "page": "https://localhost:3000/taskpane.html"
+            },
+            "lifetime": "short",
+            "actions": [
+                {
+                    "id": "TaskPaneRuntimeShow",
+                    "type": "openPage",
+                    "view": "dashboard",
+                    "pinnable": true,
+                    "supportsNoItemContext": true,
+                    "multiselect": true
```

Comment:
Do we need to explicitly configure "pinnable" and "multiselect"? From what I understand, these features are automatically enabled when "supportsNoItemContext" is set to `true`.

Reply:
It's not mandatory to include them when "supportsNoItemContext" is true, but since the default for "pinnable" and "multiselect" (when they are not present) is **false**, I think it is misleading/confusing to leave them out in circumstances where they are NOT going to have their default values.